task_schemas.md

```diff
@@ -0,0 +1,203 @@
+# BigBio Schema Documentation
+We have defined a set of lightwieght, task-specific schema to help simplify programmatic access to common biomedical datasets. This schema should be implemented for each dataset in addition to a schema that preserves the original dataset format.
+
+### Example Schemas by Task
+- [Information Extraction](#information-extraction)
+  - Named entity recognition (NER)
+  - Named entity disambiguation/normalization/linking (NED)
+  - Event extraction
+  - Relation extraction (RE)
+  - Coreference resolution
+- [Textual Entailment](#textual-entailment)
+- [Question Answering](#question-answering)
+- [Paraphasing, Translation, Summarization](#translation,-paraphasing,-summarization)
+- [Semantic Similarity](#semantic-similarity)
+- [Text Classification](#text-classification)
+
+## Information Extraction
+
+[Schema Template](schemas/kb.py)
+
+This is a simple container format with minimal nesting that supports a range of common information extraction/knowledge base construction tasks.
+
+- Named entity recognition (NER)
+- Named entity disambiguation/normalization/linking (NED)
+- Event extraction
+- Relation extraction (RE)
+- Coreference resolution
+
+```
+{
+    "document_id": "XXXXXX",
```

Comment:
we have `id` and `document_id` in the `schemas/kb.py`  maybe we can go into some detail here about how they should be used?